### PR TITLE
Initial TextBundle/TextPack support

### PR DESCRIPTION
**Pull request by nheagy**
Refs #11223

This adds basic TextBundle/TextPack import to the share extension. 

### In Bear:
- From the notes list: Swipe left → More... → Export Note... → Text Bundle
- Open in WordPress
- Ensure the contents of the post are correct, and not quoted

### In Ulysses:
- Tap share icon
- Tap the top to choose export format, choose Text and TextPack
- Tap the `…` button, tap "Send to" or "Open in Another App"
- Open in WordPress
- Ensure the contents of the post are correct, and not quoted

## Notes:
This is the preliminary support. No support for tags or images has been added yet.

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by nheagy** [comment]
@peril-wordpress-mobile this already IS the smaller PR 😂

**Comment by nheagy** [comment]
@ matteorattotti fwiw 😀

**Comment by nheagy** [comment]
@matteorattotti fwiw 😀

**Comment by matteorattotti** [comment]
Amazing! Anything I can do to help?

File: Podfile [hunk 1]
```diff
@@ -158,6 +158,7 @@ target 'WordPress' do
         shared_with_all_pods
         shared_with_networking_pods
         wordpress_ui
+        pod 'ZIPFoundation', '~> 0.9.8'
```
Comment (bummytime) [inline Podfile]: @nheagy, does `ZIPFoundation` need to be listed in the  `WordPressDraftActionExtension` extension below 👇 ?

Comment (nheagy) [reply Podfile]: I _don't think so_ but I'll try to double check.

Comment (nheagy) [reply Podfile]: It seems to build fine 😉

**Comment by nheagy** [comment]
Thanks @bummytime!

**Comment by designsimply** [comment]
Tested this today, LGTM, and wanted to add a couple settings screenshots for reference.

Bear

IMG_2511 &nbsp; IMG_2510

Ulysses

IMG_2514 &nbsp; IMG_2513

Aside: tags worked! 😍 images didn't.

**Pull request merged**